admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,115 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: how to backup and restore at project and cluster level
+:keywords: backup, restore
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+do _backup_, saving state to separate storage,
+and _restore_, recreating state from separate storage,
+at the cluster level.
+The full state of a cluster installation includes:
+- etcd data on each master
+- API objects
+//??? Anything else?
+//??? What about (non-master) nodes?
+
+[NOTE]
+Project-level backup/restore will be available in a future release.
+//??? Is this true?
```

Comment:
This should be possible today by running `oc export <object> <name> -o yaml > <name>_<object>.yaml` for all the objects that you want to save, with in a project.